lib/accesscontrol.rb

```diff
@@ -2,6 +2,10 @@
 require "accesscontrol/query"
 
 module AccessControl
+
+  unless defined?(CouldNotGrantError) == "constant" && CouldNotGrantError.class == Class
+    CouldNotGrantError = Class.new(StandardError)
+  end
   # AccessControl's tables are prefixed with access_control to
```

Comment:
Tiny nitpick, but normally I would expect to see an empty line between the `end` of one thing and the comment for the next thing.

Reply:
👍